src/app/loadout/loadout-edit/LoadoutEditSubclass.m.scss

```diff
@@ -55,3 +55,13 @@
 .missingItem {
   opacity: 0.3;
 }
+
+.canDrop {
+  filter: invert(20%);
+  background-color: rgba(255, 255, 255, 0.2);
+}
+
+.isOver {
+  filter: invert(40%);
```

Comment:
I'm having trouble imagining what this looks like...

Reply:
You should see it in the video on the description.

<img width="401" alt="Screenshot 2023-05-04 at 2 02 50 pm" src="https://user-images.githubusercontent.com/7344652/236110216-221ba4f3-8549-4dd3-b1bb-2b7348c5f379.png">